cobc/codegen.c

```diff
@@ -2598,6 +2608,10 @@ output_collating_tables (void)
 static void
 output_init_collating_tables (void)
 {
+	if (use_external_tables) {
+		return;
```

Comment:
This return looks strange, either the variables below are set - then we need those or they aren't.
If they are set but we don't need them, well, that's a different error, but should be fixed, too.

Reply:
Actually this is going to change since we're making all tables external.

We needed that early return because for external tables we embed the table in the header file (at compilation), while for builtin tables we just store a pointer (at runtime).